.tekton/integration-tests/scripts/run-console-playwright-tests.sh

```
@@ -1,28 +1,28 @@
 console_component="${1:?usage: $0 <related_images component name> <ocp channel e.g. latest-4.18>}"
 ocp_channel="${2:?usage: $0 <related_images name> <ocp channel>}"
 
 echo "COMMIT_SHA: ${COMMIT_SHA}"
-echo "CYPRESS_BASE_URL: ${CYPRESS_BASE_URL:-}"
-echo "CYPRESS_CONSOLE_IMAGE: ${CYPRESS_CONSOLE_IMAGE:-}"
+echo "BASE_URL: ${BASE_URL:-}"
+echo "CONSOLE_IMAGE: ${CONSOLE_IMAGE:-}"
 echo "---------------------------------------------"
-export CYPRESS_LOGIN_PASSWORD="$(cat "${PASSWORD_PATH}")"
-echo "(CYPRESS_LOGIN_PASSWORD set from PASSWORD_PATH; not echoed)"
+export LOGIN_PASSWORD="$(cat "${PASSWORD_PATH}")"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)